src/app/components/RelatedContentSection/index.tsx

```diff
@@ -163,7 +177,13 @@ const RelatedContentSection = ({ content }: { content: OptimoBlock[] }) => {
         {title}
       </SectionLabel>
       {hasSingleContent ? (
-        <div css={styles.singleItemWrapper}>
+        <div
+          css={[
+            headlineFirst
+              ? styles.singleItemWrapperFullWidth
+              : styles.singleItemWrapper,
+          ]}
```

Comment:
Similarly, is the [] required here?
```suggestion
          css={
            headlineFirst
              ? styles.singleItemWrapperFullWidth
              : styles.singleItemWrapper,
          }
```